lib/servers/UCLA_piezo_server/UCLA_piezo_server.py

```diff
@@ -135,7 +135,7 @@ def set_voltage(self, c, chan, voltage):
         output = 'vout.w ' + str(chan) + ' ' + str(voltage['V'])
         yield dev.write(output)
         setting = yield self.reg.get('ucla_piezo_chan_' + str(chan))
```

Comment:
I would avoid a list here, because the next lines setting[1] and setting[2] are not clear, compared to what those settings actually are called.